corehq/apps/styleguide/templates/styleguide/bootstrap5/migration_guide.html

```diff
@@ -144,44 +160,45 @@ <h2 id="split-files" class="pt-4">
     missed renamed references, especially in parts of the codebase that are undergoing frequent front-end changes. This
     initial "split file" Pull Request should <strong>only</strong> contain changes automatically made by the management command.
     Additional changes should be made in subsequent PRs.
+    <a href="https://github.com/dimagi/commcare-hq/pull/34345" target="_blank">Here is an example Pull Request</a> of this step.
   </p>
   <p>
     Use the management command below with <code>&lt;app_name&gt;</code> being the
     name of the application you would like to migrate:
   </p>
   <div class="card text-bg-light mb-3">
     <div class="card-body">
-      <pre class="m-0">./manage.py migrate_app_to_bootstrap5 &lt;app_name&gt;</pre>
+      <pre class="m-0">./manage.py migrate_app_to_bootstrap5 &lt;app_name&gt; --skip-all</pre>
     </div>
   </div>
   <p>
     For instance, if you want to migrate <code>corehq.apps.reminders</code>, you would use <code>reminders</code> as
     <code>&lt;app_name&gt;</code>.
   </p>
   <p>
-    The management command will iterate through all template and javascript files in the app. You have the option to
-    review and accept each change line-by-line or review the diff for each file at the end.
-    A changelog for each file will be saved to a markdown file in <code>BOOTSTRAP_MIGRATION_LOGS_DIR</code>.
+    The option <code>--skip-all</code> is optional, but recommended unless you have a particularly tricky app
+    as it speeds up the migration process by auto-committing the split files changes for each template and
+    javascript file in the app.
   </p>
-  <div class="alert alert-primary">
-    Due to potential merge conflict issues, the markdown files outputted to <code>BOOTSTRAP_MIGRATION_LOGS_DIR</code>
-    are not version controlled. If you want to work on migrating an application with multiple people, you can zip
-    these markdown files and share with your collaborators for future reference.
-  </div>
   <p>
-    Once a file is updated and split into the two bootstrap versions, the command will
-    find references to the original filepath and replace them with the filepath to the <code>bootstrap3</code> version.
+    If <code>--skip-all</code> is not used, the management command will iterate through all template and javascript
+    files in the app. You have the option to review and accept each change line-by-line or review
+    the diff for each file at the end.
   </p>
   <p>
-    After applying all the actions related to splitting a file, it is recommended to commit those changes before
-    moving on to the next file. The management command will prompt you to do this with a suggested commit message.
-    Formatting our commits this way is important to maintain a clean git history.
+    Regardless of what route you take, a changelog for each file will be saved to a markdown file in
```

Comment:
Idle thought: I would *love* to write one changelog for automated fixes and one for flagged issues (and will probably make that update eventually). I start every view migration by editing the change log, which is hundreds of lines long even for the small migrations I've been doing, and I either delete all the automated changes and re-save the log, or I make TODO comments in the code for the flagged issues and then work off of those and forget about the log file.

Reply:
ah, i do like this thought. how about adding `#todo` comments around flagged areas as well? I think that would be helpful too

Reply:
would you like me to include these improvements in this PR or a followup?

Reply:
TODO comments would be amazing. Definitely doesn't need to be part of this PR.